packages/teams-js/test/private/notifications.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { NotificationTypes, ShowNotificationParameters } from '../../src/private/interfaces';
+import { notifications } from '../../src/private/notifications';
+import { app } from '../../src/public/app';
+import { errorNotSupportedOnPlatform, FrameContexts } from '../../src/public/constants';
+import { Utils } from '../utils';
+
+const allowedContexts = [FrameContexts.content];
+describe('notifications', () => {
+  const utils = new Utils();
+  beforeEach(() => {
+    utils.processMessage = null;
+    utils.messages = [];
+    utils.childMessages = [];
+    utils.childWindow.closed = false;
+  });
+
+  afterEach(() => {
+    // Reset the object since it's a singleton
+    if (app._uninitialize) {
+      app._uninitialize();
+    }
+  });
+
+  describe('showNotification', () => {
+    const showNotificationParameters: ShowNotificationParameters = {
+      message: 'Some Message',
+      notificationType: NotificationTypes.fileDownloadStart,
+    };
+    it('should not allow calls before initialization', () => {
+      expect(() => notifications.showNotification(showNotificationParameters)).toThrowError(
+        'The library has not yet been initialized',
+      );
+    });
+    Object.values(FrameContexts).forEach(context => {
+      if (allowedContexts.some(allowedContexts => allowedContexts === context)) {
+        it(`showNotification should throw error if notifications capability is not supported in runtime config. context: ${context}`, async () => {
+          await utils.initializeWithContext(context);
+          expect.assertions(1);
```

Comment:
why the expected assertions is just 1 /Are we not counting the assertions in initializeWithContext?

Reply:
This one is just for catch block to be executed and not for initializeWithContext